installing/installing_rhv/installing-rhv-restricted-network.adoc

```diff
@@ -0,0 +1,87 @@
+[id="installing-rhv-restricted-network_{context}"]
+= Installing a cluster on {rh-virtualization} in a restricted network
+include::modules/common-attributes.adoc[]
+:context: installing-rhv-restricted-network
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a
+customized {product-title} cluster on {rh-virtualization-first} in a restricted network by creating an internal mirror of the installation release content.
+
+== Prerequisites
+
+The following items are required to install an {product-title} cluster on a {rh-virtualization} environment.
+
+* You have a supported combination of versions in the link:https://access.redhat.com/articles/5485861[Support Matrix for {product-title} on {rh-virtualization}].
+* You are familiar with the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Create a registry on your mirror host] and obtain the `imageContentSources` data for your version of {product-title}.
++
+[IMPORTANT]
+====
+Because the installation media is on the mirror host, you can use that computer
+to complete all installation steps.
+====
+* Provision
+xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage]
+for your cluster. To deploy a private image registry, your storage must provide
+ReadWriteMany access modes.
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall and plan to use telemetry, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure the firewall to allow the sites] that your cluster requires access to.
```

Comment:
Can you remove the hard wraps on these lines?

Reply:
wasn't aware that I made those - possibly copy/paste artifacts...